x-pack/test_serverless/functional/test_suites/search/cases/attachment_framework.ts

```diff
@@ -34,7 +34,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
           'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
         );
 
-        await settings.refreshDataViewFieldList('default:all-data');
+        await settings.refreshDataViewFieldList('default:all-data', { ignoreMissing: true });
```

Comment:
I think the solution is good: we explicitly state missing data views are acceptable.